farcasterapi/neynar/neynar.go

```diff
@@ -295,8 +297,19 @@ func (n *NeynarAPI) ChannelFIDs(ctx context.Context, channelID string) ([]uint64
 	if !exists {
 		return nil, farcasterapi.ErrChannelNotFound
 	}
+	// get the followers of the channel to update the progress
+	channelURL := fmt.Sprintf(warpcastChannelInfo, channelID)
+	body, err := n.request(channelURL, http.MethodGet, nil, defaultRequestTimeout)
+	if err != nil {
+		return nil, fmt.Errorf("error creating request: %w", err)
```

Comment:
We should consider that the warpcast api could fail, change or ban us. So we cannot make this dependent to that. If the call fails, there should be a fallback.